src/main/webapp/task.js

```diff
@@ -25,12 +25,12 @@ class Task {
    *
    * @param {string} title a title for the task
    * @param {string} notes messages associated with a given task
-   * @param {Date} due a Date object / null if not present
+   * @param {Date} due a Date object / undefined if not present
    */
   constructor(title, notes, due) {
     this.title = title;
     this.notes = notes;
-    if (due !== null) {
+    if (due != null) {
```

Comment:
`!==` for consistency

Reply:
`!==` and `!=` are actually different operators, where `!=` allows for implicit type coercion and `!==` does not. Fun fact: this is why `true == 1` returns `true` but `true === 1` returns false. 

Usually `!=` is frowned upon, except for catching both `null` and `undefined` values, which is what I'm using this for here. https://google.github.io/styleguide/jsguide.html#features-equality-checks-exceptions